### PR TITLE
chore(github-ci): Add new strategies for simple-book-only

### PR DESCRIPTION
**Pull request by lukehesluke**
I was going to look into changing the validator so that it only validated against a feature if every test identifier in that feature was guarantueed to be included given the config, but in the end, it's simpler to just turn off minimal-proposal for simple booking flow 🤷  Otherwise, this task would have been quite difficult and made various sections even more complicated

The same should also be doable for approval-flow-only, though this will depend on the current state of `supportsAvailable: true`

### CODE REVIEW

File: packages/openactive-integration-tests/test/features/approval/minimal-proposal/feature.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
   "description": "Test Minimal Proposal Implementation approval flows. OrderProposal Amendments are not included here.",
   "explainer": "",
   "specificationReference": "https://openactive.io/open-booking-api/EditorsDraft/#minimal-proposal-implementation",
-  "required": true,
+  "required": false,
```
Comment (lukehesluke) [inline packages/openactive-integration-tests/test/features/approval/minimal-proposal/feature.json:8]: not required for simple booking

File: packages/openactive-integration-tests/test/features/core/opportunity-feed/feature.json [hunk 2]
```diff
@@ -5,8 +5,8 @@
   "description": "Real-time opportunity data",
   "explainer": "",
   "specificationReference": "https://www.openactive.io/open-booking-api/EditorsDraft/#definition-of-a-bookable-opportunity-and-offer-pair",
-  "required": true,
-  "coverageStatus": "complete",
+  "required": false,
+  "coverageStatus": "none",
```
Comment (lukehesluke) [inline packages/openactive-integration-tests/test/features/core/opportunity-feed/feature.json:9]: There are no tests here, so I set this as such

File: packages/openactive-integration-tests/test/reporter.js [hunk 3]
```diff
@@ -179,19 +179,28 @@ class Reporter {
         const html = await certificationWriter.generateCertificate();
 
         const validationResult = await validateCertificateHtml(html, CONFORMANCE_CERTIFICATE_ID, certificationWriter.awardedTo.name);
-        if (!validationResult || !validationResult.valid) {
+        /* process.env.DEBUG_SAVE_INVALID_CONFORMANCE_CERTIFICATE can be used to ensure a certificate is saved even if it
+        is not valid. This can help with debugging conformance certificates */
+        if (validationResult?.valid || process.env.DEBUG_SAVE_INVALID_CONFORMANCE_CERTIFICATE === 'true') {
```
Comment (lukehesluke) [inline packages/openactive-integration-tests/test/reporter.js:184]: Very helpful to me for debugging

File: packages/openactive-integration-tests/test/reporter.js [hunk 4]
```diff
@@ -179,19 +179,28 @@ class Reporter {
...
+          const filename = 'index.html';
+          await mkdirp(certificationWriter.certificationOutputPath);
+          await fs.writeFile(certificationWriter.certificationOutputPath + filename, html);
+          if (validationResult?.valid) {
+            console.log(`\n${chalk.green(
+              `Conformance certificate for '${certificationWriter.awardedTo.name}' generated successfully: ${certificationWriter.certificationOutputPath + filename} and must be made available at '${CONFORMANCE_CERTIFICATE_ID}' to be valid.`,
+            )}`);
+          } else {
+            console.log(`\n${chalk.red(
+              `Conformance certificate not valid but saved anyway to: ${certificationWriter.certificationOutputPath + filename}`,
```
Comment (nickevansuk) [inline packages/openactive-integration-tests/test/reporter.js:194]: Perhaps better not to leave the code for generating invalid certificates in here, as is open to abuse?

Comment (lukehesluke) [reply packages/openactive-integration-tests/test/reporter.js:194]: Is there another way to debug certificate validation without having to fully run all the tests in debug mode?

Comment (lukehesluke) [reply packages/openactive-integration-tests/test/reporter.js:194]: Also, what kind of abuse do you mean @nickevansuk ? The code is open source anyway

**Comment by lukehesluke** [comment]
The CI fails are due to https://github.com/openactive/OpenActive.Server.NET/issues/127 & https://github.com/openactive/openactive-test-suite/issues/392 so I consider this mergeable

File: packages/openactive-integration-tests/test/reporter.js (hunk 4 above)
Comment (nickevansuk) [reply packages/openactive-integration-tests/test/reporter.js:194]: Good point!

**Review by nickevansuk — APPROVED** [review]

**Pull request merged**